src/main/java/io/apicurio/datamodels/cmd/commands/RenameSchemaDefinitionCommand.java

```diff
@@ -46,7 +48,8 @@
 
     public String _oldName;
     public String _newName;
-    public List<NodePath> _references;
+    public List<NodePath> _directReferences;
```

Comment:
We need to keep the name of `_references` the same so that existing, persisted commands of this type will continue working.  If we rename the variable, then we'll potentially break old servers (anything with one of these commands persisted in the DB but not yet applied to a snapshot).

Bottom line:  let's change the name of `_directReferences` back to _references`